db/migrate/20231023191506_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,21 @@
+MIGRATION_CLASS =
+  if ActiveRecord::VERSION::MAJOR >= 5
+    ActiveRecord::Migration["#{ActiveRecord::VERSION::MAJOR}.#{ActiveRecord::VERSION::MINOR}"]
+  else
+    ActiveRecord::Migration
+  end
```

Comment:
Can you explain if/why this is necessary?

Reply:
When installing the friendy_id gem this file was generated, I believe this can be removed.

Reply:
Removed